[2025-04-24T14:47:42.392+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-24T14:47:42.399+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: airflow_lab.filter_new_data manual__2025-04-24T14:47:33.913776+00:00 [queued]>
[2025-04-24T14:47:42.403+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: airflow_lab.filter_new_data manual__2025-04-24T14:47:33.913776+00:00 [queued]>
[2025-04-24T14:47:42.403+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-04-24T14:47:42.408+0000] {taskinstance.py:2888} INFO - Executing <Task(_PythonDecoratedOperator): filter_new_data> on 2025-04-24 14:47:33.913776+00:00
[2025-04-24T14:47:42.411+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=183) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-04-24T14:47:42.412+0000] {standard_task_runner.py:72} INFO - Started process 185 to run task
[2025-04-24T14:47:42.412+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', '***_lab', 'filter_new_data', 'manual__2025-04-24T14:47:33.913776+00:00', '--job-id', '12', '--raw', '--subdir', 'DAGS_FOLDER/elt.py', '--cfg-path', '/tmp/tmphw78ufay']
[2025-04-24T14:47:42.413+0000] {standard_task_runner.py:105} INFO - Job 12: Subtask filter_new_data
[2025-04-24T14:47:42.434+0000] {task_command.py:467} INFO - Running <TaskInstance: airflow_lab.filter_new_data manual__2025-04-24T14:47:33.913776+00:00 [running]> on host 7ce975cba66d
[2025-04-24T14:47:42.477+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='***_lab' AIRFLOW_CTX_TASK_ID='filter_new_data' AIRFLOW_CTX_EXECUTION_DATE='2025-04-24T14:47:33.913776+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-24T14:47:33.913776+00:00'
[2025-04-24T14:47:42.479+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-04-24T14:47:42.480+0000] {logging_mixin.py:190} INFO - No API data to filter.
[2025-04-24T14:47:42.480+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-04-24T14:47:42.484+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-04-24T14:47:42.485+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=***_lab, task_id=filter_new_data, run_id=manual__2025-04-24T14:47:33.913776+00:00, execution_date=20250424T144733, start_date=20250424T144742, end_date=20250424T144742
[2025-04-24T14:47:42.514+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-24T14:47:42.525+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-04-24T14:47:42.526+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
